NSS-Day-Cohort-69/tabloid-hamburgers
# Ft/tags view

Purpose: Allows Admin to view all tags

Files Changed:
Controllers/TagController.cs
Migrations/TabloidDbContextModelSnapshot.cs
Models/DTOs/TagDTO.cs
client/src/components/ApplicationViews.jsx
client/src/components/NavBar.jsx
client/src/components/tags/TagsView.jsx
client/src/managers/tags.js

To Test: 
While as an Admin, check if the Navbar allows you too view the Tag List link, and Tag List component.

Closes: #34
